docs/migration.md

```diff
@@ -145,23 +145,27 @@ The following properties have been removed from or changed in `Field`:
 
 ### Changes to dataclasses
 
+Pydantic [dataclasses](usage/dataclasses.md) continue to be useful for enabling the data validation on standard
+dataclasses without having to subclass `BaseModel`. However, we have made some changes to the behavior of Pydantic
+dataclasses in V2:
```

Comment:
I haven't seen `we` used in the docs yet – perhaps change language to be more subjective.

```suggestion
Pydantic [dataclasses](usage/dataclasses.md) continue to be useful for enabling the data validation on standard
dataclasses without having to subclass `BaseModel`. Pydantic V2 introduces the following changes to this dataclass behavior:
```

Reply:
Oh, lol, I do see `we` used... Welp.